packages/datepicker/package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "@aurelia-ux-components/datepicker",
+  "version": "0.0.1",
+  "description": "Checkbox component for Aurelia UX",
```

Comment:
Make sure to check all package metadata. I just noticed this one isn't quite right.